notebooks/MAST/plot_analyze_with_lightkurve/plot_analyze_with_lightkurve.ipynb

```diff
@@ -0,0 +1,953 @@
+{
```

Comment:
<p>Note for me to replace "lightcurve" everywhere with "light curve" as per comments on pull request 12.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>